.github/workflows/lint.yml

```diff
@@ -24,18 +24,13 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "pip"
```

Comment:
By the way, I think these caches are now outdated.

Maybe something for a follow up though!